packages/family-tree/src/NodeAnalyzer/PropertyUsageAnalyzer.php

```diff
@@ -75,8 +75,8 @@ public function isPropertyFetchedInChildClass(Property $property): bool
 
         $childrenClassReflections = $this->familyRelationsAnalyzer->getChildrenOfClassReflection($classReflection);
 
-        foreach ($childrenClassReflections as $childClassReflection) {
-            $childClass = $this->nodeRepository->findClass($childClassReflection->getName());
+        foreach ($childrenClassReflections as $childrenClassReflection) {
```

Comment:
Children to child was correct here, should be kept

Reply:
any way to singularize except using Inflector service for variable?

Reply:
Nothing I'm aware of. Maybe current Inflector will need modification to cover children/child prefix

Reply:
I will try add it.

Reply:
implemented.